spec/features/user_can_add_reviews_spec.rb

```diff
@@ -18,3 +18,22 @@
     )
   end
 end
+
+feature 'reviews paginate' do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'reviews show on multiple pages' do
+    sign_in_as(user)
+    8.times { make_review_for_game(game) }
```

Comment:
You can also create a list of games with FactoryGirl. Look under "Multiple Records" here:

https://horizon.launchacademy.com/lessons/factory-girl